src/components/application_manager/src/commands/mobile/alert_maneuver_request.cc

```diff
@@ -207,10 +207,16 @@ bool AlertManeuverRequest::PrepareResponseParameters(
     return_info = std::string("Unsupported phoneme type sent in a prompt");
     return result;
   }
+
   result_code =
       PrepareResultCodeForResponse(navigation_alert_info, tts_alert_info);
   return_info =
       MergeInfos(navigation_alert_info, info_navi_, tts_alert_info, info_tts_);
+
+  if (mobile_apis::Result::UNSUPPORTED_RESOURCE == result_code) {
+    // According to APPLINK-19582 should be success:true in this case
```

Comment:
@AKalinich-Luxoft Remove JIRA task from comment. It is prohibited to specify such things in OpenSDL.